.gitignore

```diff
@@ -9,3 +9,5 @@ crashes
 filter.json
 whitelist.txt
 action_remove_mass_followers.py
+env
```

Comment:
Ignore these files locally in your PC, not in the project's .gitignore.

Reply:
Sorry, I'm new to github.. what do you mean "ignore these files locally"? I'm using GitHub desktop and found only that solution to remove these 540 files

Reply:
https://stackoverflow.com/questions/1753070/how-do-i-configure-git-to-ignore-some-files-locally